src/borg/fuse.py

```diff
@@ -340,7 +340,7 @@ def read(self, fh, offset, size):
                     # evict fully read chunk from cache
                     del self.data_cache[id]
             else:
-                data = self.key.decrypt(id, self.repository.get(id))
+                data = self.key.decrypt(id, self.repository_uncached.get(id))
```

Comment:
this could be a large performance regression for specific usecases. It's unfortunate to hide this in a mildly related pull request.